docs/gambit/mutation-verifier.md

```diff
@@ -35,129 +34,205 @@ If you are on Linux, you will need pip version 20.3 or above.
+  specific changes that must always be checked, 
+  or mutations that Gambit does not currently support.
+
+```{note}
+It is recommended to limit each manually mutated file to a single mutation for more accurate coverage analysis and better traceability.
+```
+
+
+To add manual mutations to `mutation.mconf`, write:
 
 ```json
-{ "gambit": [{
+{ 
+  "gambit": [{
     "filename" : "C.sol",
     "num_mutants": 5
   }],
   "manual_mutants": {
-     "C.sol": "path/to/dir/with/manual_mutants/for/C"
+     "C.sol": "path/to/dir/with/manual_mutants/for/C.m1.sol"
   }
 }
 ```
-If you set `num_mutants` to 0 in the above file, `gambit` will run only on the mutants added manually and no other mutants will be generated.
-`certoraMutate` on manually written mutants.
+If you don't have a `gambit` object in the `mconf` file, 
+  `certoraMutate` will run only on the manual mutants, 
+  and no other mutants will be generated.
 
 ```{note}
-All manual mutations must be named uniquely. For example, if you want to generate manual mutations for `C.sol` and `D.sol`, name them as `C.m1.sol, C.m2.sol, D.m3.sol, D.m4.sol, ...` etc.
+All manual mutations must be named uniquely. 
+For example, if you want to generate manual mutations for `C.sol` and `D.sol`, 
+  name them `C.m1.sol, C.m2.sol, D.m3.sol, D.m4.sol, ...` etc.
 ```
 
-## CLI Options
+## Original Verification Run
+
+A mutation test requires an original verification job that was completed successfully without halting. All mutant checks will be run with the same verification configuration as the original run, 
+  and their results will be compared to the original run. 
+Rules that are not verified on the original run will be ignored.
+
+(orig-verification-link)=
+### Original Verification Link
 
-`certoraMutate` runs in two distinct modes: synchronous and asynchronous.
-Use the `--sync` flag to run the entire tool synchronously
-in your shell, from mutant generation to the web report UI. 
-Alternatively, running without the `--sync` flag will dump
-data about the mutation verification jobs in the `collect.json` file in the working directory. These jobs are submitted
-to the server environment specified and run asynchronously. 
-They may be polled later with
-`certoraMutate --collect_file collect.json`.
+You can provide the original run job's link via `--orig_run`, for example: 
+```sh
+--orig_run https://prover.certora.com/output/53342/9487899b2afc4709899889fab6c2c673/?anonymousKey=5c365717c9c1076f0c1acb050c7eb5867f07a236
+```
+
+```{note}
+The run must have the job status `Executed` on the [Prover dashboard](https://prover.certora.com).
+```
+
+The files will be downloaded to either a default directory or to one specified with `--orig_run_dir`.
+
+(prover-conf)=
+### Prover Configuration
+
+Alternatively, you can provide a verification configuration file for the Certora Prover 
+  via `--prover_conf`. 
+If you do, the Prover will execute the verification job using this file on the 
```

Comment:
will **first** execute the verification

Reply:
It will be done in parallel if you are not running locally